test/formatCurrency.spec.js

```diff
@@ -0,0 +1,24 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatCurrency from "../dist/currency";
+
+Globalize.load(
+    require( "cldr-data/main/en/numbers" ),
+    require( "cldr-data/main/en/currencies" ),
+    require( "cldr-data/supplemental/currencyData" ),
+    require( "cldr-data/supplemental/likelySubtags" )
+);

```

Comment:
I assume the specificity of CLDR content isn't subject to react-globalize tests goal, so I would simply replace these with:

```js
Globalize.load(
  require( "cldr-data" ).entireSupplemental(),
  require( "cldr-data" ).entireMainFor( "en" )
);
```

Ideally, we could place this in a `before` step for all tests and don't need to write this in every files. :)